datatypes/papas/Path.h

```diff
@@ -0,0 +1,61 @@
+//
+//  Created by Alice Robson on 02/12/15.
+//
+//
+#ifndef path_h
+#define path_h
+
+#include "Definitions.h"
+#include "TLorentzVector.h"
+#include "TVector3.h"
+#include <map>
+#include <memory>
+#include <vector>
+
+namespace papas {
+
+/// Path followed by a particle in 3D space.
+///
+/// Assumes constant speed magnitude both along the z axis and in the transverse plane.
+/// Path base class is essentially straightline but can be inherited from to make Helix etc
+///
+class Path {
```

Comment:
Why no StraightLine class? 
I think that Path should be an abstract base class.